docs/administration-guide/security/user-control.md

```diff
@@ -1,3 +1,49 @@
+# User Authentication
+
 !!! info "Work in Progress"
 
-    This page is under construction.
\ No newline at end of file
+    This page is under construction.
+
+The security layer for Gaffer is currently only enforced by the REST API. We
+recommend restricting users such that they do not have access to the cluster and
+by only allowing users to connect to Gaffer via the REST API.
+
+In the REST API the `User` object is constructed via a [`UserFactory`](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/rest/factory/UserFactory.html).
+Under the Spring REST API this has an abstract implementation class
+`AbstractUserFactory` to allow passing HTTP headers to be used for
```

Comment:
```suggestion
`AbstractUserFactory`, which is then used in the passing of HTTP headers for
```

Think these comments make it a bit clearer what you are trying to say here. Feel free to re-write however though